find_email_addresses.py

```diff
@@ -0,0 +1,45 @@
+import sys, re
+import link_extractor as le
+from collections import deque
+
+debug = False
+
+try:
+	domain = sys.argv[1]
+except:
+	domain = 'web.mit.edu'
```

Comment:
The main purpose of this was testing, it was easier to just build through sublime and check the output on a default domain than to use the command line input. The entire CLI should be update to verify that the input is correct, as well as deal with debug/help tags.